src/Components/Notifications/NotificationViewingRoom.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react"
+import { graphql, useFragment } from "react-relay"
+import { RouterLink, RouterLinkProps } from "System/Router/RouterLink"
+import { NotificationViewingRoom_viewingRoom$key } from "__generated__/NotificationViewingRoom_viewingRoom.graphql"
+import { AuthContextModule } from "@artsy/cohesion"
+import { Box, Button, Image, Spacer, Text } from "@artsy/palette"
+import { resized } from "Utils/resized"
+
+const MAX_WIDTH = 600
```

Comment:
If we are reusing this value, do you think we could move it to one of the parent components?

Reply:
@dariakoko yes, I already did in a separate PR here: https://github.com/artsy/force/pull/13456/files#diff-edd4e49eedcd30401192b266b9596e84735f5fbf6bb8f30ca09871a0bbbc9fa7R1